### PR TITLE
test: avoid clearing auth token in e2e tests [DET-4486]

### PR DESCRIPTION
**Pull request by hamidzr**
## Checklist

- [x] User-facing API changes need the "User-facing API Change" label.
- [x] Release notes should be added as a separate file under `docs/release-notes/`.
See [Release Note](../docs/release-notes/README.md) for details.

[DET-1234]: https://determinedai.atlassian.net/browse/DET-1234

### CODE REVIEW

**Comment by hamidzr** [comment]
https://app.circleci.com/pipelines/github/determined-ai/determined/7140/workflows/5cce0551-8560-45ab-937d-43b5ecd29c78

File: webui/tests/cypress/integration/02-navigation.spec.ts [hunk 1]
```diff
@@ -9,6 +12,10 @@ describe('Navigation', () => {
 
   describe('paths', () => {
 
+    beforeEach(() => {
+      cy.restoreLocalStorageCache([ STORAGE_KEY_AUTH ]);
+    });
+
```
Comment (hkang1) [inline webui/tests/cypress/integration/02-navigation.spec.ts:18]: **suggestion:** move this outside of `describe('paths', ...)` so you don't have to duplicate `restoreLocalStorageCache` for `describe('side menu buttons', ...)`?

File: webui/tests/cypress/integration/02-navigation.spec.ts [hunk 2]
```diff
@@ -49,6 +56,7 @@ describe('Navigation', () => {
 
   describe('side menu buttons', () => {
     beforeEach(() => {
+      cy.restoreLocalStorageCache([ STORAGE_KEY_AUTH ]);
       cy.visit('/det');
```
Comment (hkang1) [inline webui/tests/cypress/integration/02-navigation.spec.ts:60]: **thought:** should we take this out or only apply it one time via `before()`? With the idea that all the subsequent navigation menu should work no matter what page you are on.

**Review by hkang1 — APPROVED** [review]
LGTM. Couple of comments, all of them non-blocking.

File: webui/tests/cypress/integration/02-navigation.spec.ts (hunk 1 above)
Comment (hamidzr) [reply webui/tests/cypress/integration/02-navigation.spec.ts]: no need anymore https://github.com/determined-ai/determined/pull/1539/commits/69aaa838349fe2e27f7d966d9822c8e36d40bd9e

**Comment by hamidzr** [comment]
webui passed. I'm gonna push one last change. https://circleci.com/gh/determined-ai/determined/189700?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-checks-link

**Pull request merged**